src/test/java/core/basesyntax/basesyntax/service/impl/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.basesyntax.service.impl;
+
+import core.basesyntax.basesyntax.db.Storage;
+import core.basesyntax.basesyntax.model.dto.FruitRecordDto;
+import core.basesyntax.basesyntax.service.OperationHandler;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private OperationHandler balanceOperationHandler = new BalanceOperationHandler();
+
+    @AfterClass
+    public static void afterClass() throws Exception {
```

Comment:
since you single test method it's not obligatory to have after, but if you want to have it here, I suggest you mark as After instead of AfterClass

Reply:
done